### ISSUE TITLE
Poller expiration

### ISSUE THREAD

**Issue by tbranyen** [bocoup/email#5]
It should be possible to have once-like pollers that terminate after successful.

**Comment by tbranyen** [comment]
Pollers that specify an exact date to run on will expire, versus cron helpers.  This is ideal because it's the exact same API.

Poll every day:

``` json
{
  "schedule": "@daily"
}
```

Schedule a one time job in 5 seconds:

``` json
{
  "schedule": Date.now() * 5000
}
```

**Comment by tafsiri** [comment]
would daily let you specify a time for when in the day?

**Comment by tbranyen** [comment]
No, it'll correlate directly to the Cron format indicating a day.

**Comment by tafsiri** [comment]
would this api basically support my sending any cron string, e.g. 30 2 * * *?

**Comment by tbranyen** [comment]
Yup, using the node-crontab module, that will work out-of-the-box.
